build/charts/antrea/templates/crds/traceflow.yaml

```diff
@@ -189,6 +189,10 @@ spec:
                               type: string
                             tunnelDstIP:
                               type: string
+                            egressIP:
+                              type: string
+                            egressPolicy:
```

Comment:
In Egress API itself, we don't call it EgressPolicy, and EgressPolicy may be confusing with the Egress NetworkPolicy rule. It should just be Egress.